ext/fg/js/frame-offset-forwarder.js

```diff
@@ -91,6 +93,53 @@ class FrameOffsetForwarder {
         this._forwardFrameOffset(offset, uniqueId);
     }
 
+    _findFrameWithContentWindow(contentWindow) {
+        const elementSources = [
```

Comment:
97-116 can be simplified by using a generator.

```js
function* getElementSources() {
  yield [...this._frameCache];
  yield [...document.querySelectorAll('frame, iframe:not(.yomichan-float)')];
  yield [document.documentElement];
}

const ELEMENT_NODE = Node.ELEMENT_NODE;
for (const elements of getElementSources()) {
  while (elements.length > 0 || elements.push(...getMoreElements())) {
    // ...
```

Reply:
https://github.com/FooSoft/yomichan/pull/460/commits/08ded3154f45d23687e310800b6b7b4eb2133aef